source/sema.h

```diff
@@ -502,228 +482,36 @@ class sema
 
         }
 
-        //--------- START TEMPORARY REGRESSION TEST CODE FOR G_D_O OPTIMIZATION VERIFICATION ---------
-        //  Now do a regression test violation check
-        //
-        if (
-            flag_internal_debug
-            && result_old != result
-            )
-        {
-            std::cerr << "\n  Internal compiler error - see cppfront-ice-data.out for debug information\n\n";
+        ////--------- START TEMPORARY REGRESSION TEST CODE FOR G_D_O OPTIMIZATION VERIFICATION ---------
+        ////  Now do a regression test violation check
+        ////
+        //if (
+        //    flag_internal_debug
+        //    && result_old != result
+        //    )
+        //{
+        //    std::cerr << "\n  Internal compiler error - see cppfront-ice-data.out for debug information\n\n";
 
-            auto out = std::ofstream{"cppfront-ice-data.out"};
+        //    auto out = std::ofstream{"cppfront-ice-data.out"};
 
-            out << "g_d_o arguments:\n";
-            out << "    " << static_cast<void const*>(&t) << " -> " << t.as_string_view() << " @ " << t.position().to_string()
-                << ", token order # " << t.get_global_token_order() << "\n";
-            out << "    look_beyond_current_function: " << std::boolalpha << look_beyond_current_function << "\n";
-            out << "    include_implicit_this:        " << std::boolalpha << include_implicit_this        << "\n";
+        //    out << "g_d_o arguments:\n";
+        //    out << "    " << static_cast<void const*>(&t) << " -> " << t.as_string_view() << " @ " << t.position().to_string()
+        //        << ", token order # " << t.get_global_token_order() << "\n";
+        //    out << "    look_beyond_current_function: " << std::boolalpha << look_beyond_current_function << "\n";
+        //    out << "    include_implicit_this:        " << std::boolalpha << include_implicit_this        << "\n";
 
-            out << "result_old:    "       << static_cast<void const*>(result_old) << "\n";
-            out << "result:        "       << static_cast<void const*>(result    ) << "\n\n";
+        //    out << "result_old:    "       << static_cast<void const*>(result_old) << "\n";
+        //    out << "result:        "       << static_cast<void const*>(result    ) << "\n\n";
 
-            debug_print(out);
+        //    debug_print(out);
 
-            exit(EXIT_FAILURE);
-        }
-        //--------- END TEMPORARY REGRESSION TEST CODE FOR G_D_O OPTIMIZATION VERIFICATION -----------
+        //    exit(EXIT_FAILURE);
+        //}
+        ////--------- END TEMPORARY REGRESSION TEST CODE FOR G_D_O OPTIMIZATION VERIFICATION -----------
```

Comment:
I guess this was commented out for testing and can be removed now?

```suggestion
```

Reply:
Thanks! Yes I just forgot to remove that part. It shouldn't affect regression tests though. 👍